ddtrace/ext/kombu.py

```diff
@@ -0,0 +1,10 @@
+# type of the spans
+TYPE = 'kombu'
+
+# net extension
+VHOST = 'out.vhost'
```

Comment:
If you use `out.host`, our system will detect the tag and consider the execution as happened in another host (assuming the host is really different). There is a reason why you prefer `out.vhost`? Maybe it's part of the Kombu vocabulary?

Reply:
A vhost is not a physical host, but a string that is part of the rabbit url. This is typically used to have multiple tenants in a rabbit cluster, say if you had two logically distinct apps that wanted to save on hardware costs. The deafult is `/` (urlencoded `%2F`), and this is typically the default.